xlators/mgmt/glusterd/src/glusterd-mgmt.c

```diff
@@ -2129,6 +2129,8 @@ glusterd_mgmt_v3_post_validate(glusterd_op_t op, int32_t op_ret, dict_t *dict,
     ret = gd_mgmt_v3_post_validate_fn(op, op_ret, req_dict, op_errstr,
                                       rsp_dict);
 
+    dict_unref(rsp_dict);
+    rsp_dict = NULL;
```

Comment:
This is not really needed. `rsp_dict` is not used anymore.

Reply:
Ack